as/plugins/org.jboss.tools.as.rsp.ui/src/org/jboss/tools/as/rsp/ui/RspUiActivator.java

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.as.rsp.ui;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.jboss.tools.foundation.core.plugin.log.IPluginLog;
+import org.jboss.tools.foundation.core.plugin.log.StatusFactory;
+import org.jboss.tools.foundation.ui.plugin.BaseUIPlugin;
+import org.jboss.tools.foundation.ui.plugin.BaseUISharedImages;
+import org.jboss.tools.usage.event.UsageEventType;
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+
+public class RspUiActivator extends BaseUIPlugin {
+
+	// The plug-in ID
+	public static final String PLUGIN_ID = "org.jboss.tools.rsp.ui"; //$NON-NLS-1$
+
+	// Usage detection
+	public static final String USAGE_COMPONENT_NAME = "server"; //$NON-NLS-1$
+
+	// The shared instance
+	private static RspUiActivator plugin;
+
+	private UsageEventType newServerEventType;
+	private UsageEventType newRemoteServerEventType;
+
+	/**
+	 * The constructor
+	 */
+	public RspUiActivator() {
+	}
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see org.eclipse.ui.plugin.AbstractUIPlugin#start(org.osgi.framework.
+	 * BundleContext)
+	 */
+	public void start(BundleContext context) throws Exception {
+		super.start(context);
+		plugin = this;
+
+//		newServerEventType = new UsageEventType(USAGE_COMPONENT_NAME, UsageEventType.getVersion(ASWTPToolsPlugin.this), 
```

Comment:
please enable those metrics events, it will be available in amplitude for a future chart.